bindings/python/src/cloudsmith_api/models/entitlement_usage_metric.py

```diff
@@ -31,75 +31,46 @@ class EntitlementTokenMetric(object):
                             and the value is json key in definition.
     """
     swagger_types = {
-        'bandwidth_per_token': 'object',
-        'totals': 'object'
+        'usage': 'object'
     }
 
     attribute_map = {
-        'bandwidth_per_token': 'bandwidth_per_token',
-        'totals': 'totals'
+        'usage': 'usage'
     }
 
-    def __init__(self, bandwidth_per_token=None, totals=None):
+    def __init__(self, usage=None):
         """
-        EntitlementTokenMetric - a model defined in Swagger
+        EntitlementUsageMetric - a model defined in Swagger
         """
 
-        self._bandwidth_per_token = None
-        self._totals = None
+        self._usage = None
 
-        self.bandwidth_per_token = bandwidth_per_token
-        self.totals = totals
+        self.usage = usage
 
     @property
-    def bandwidth_per_token(self):
+    def usage(self):
         """
-        Gets the bandwidth_per_token of this EntitlementTokenMetric.
+        Gets the usage of this EntitlementUsageMetric.
         
 
-        :return: The bandwidth_per_token of this EntitlementTokenMetric.
+        :return: The usage of this EntitlementUsageMetric.
         :rtype: object
         """
-        return self._bandwidth_per_token
+        return self._usage
 
-    @bandwidth_per_token.setter
```

Comment:
Hmmm. These changes will break the CLI, I think.

Reply:
I suppose it's already broken since the data we output might be different ...

Reply:
Yeah. It's already broken. It's why I was keen to try and get these changes out in order to update the CLI and release a fixed version.